config.js

```diff
@@ -59,10 +59,12 @@ module.exports = {
     // Refer to https://github.com/freyacodes/Lavalink to host your own lavalink server
     LAVALINK_NODES: [
       {
-        host: "localhost",
-        port: 2333,
-        password: "youshallnotpass",
-        id: "Local Node",
+        host: "lava4.horizxon.studio",
```

Comment:
This was meant to just be a placeholder.
We are not going to promote any lavalink server. So revert this part